packages/main/src/DayPicker.ts

```diff
@@ -446,6 +459,22 @@ class DayPicker extends CalendarPart implements ICalendarPicker {
 		return timestamp === this.selectedDates[0] || timestamp === this.selectedDates[this.selectedDates.length - 1];
 	}
 
+	_isRangeEndDate(timestamp: number): boolean {
+		if (this.selectionMode === CalendarSelectionMode.Range) {
+			return timestamp === this.selectedDates[1];
+		}
+
+		return false;
```

Comment:
Just a suggestion: 
return this.selectionMode === CalendarSelectionMode.Range && timestamp === this.selectedDates[1]